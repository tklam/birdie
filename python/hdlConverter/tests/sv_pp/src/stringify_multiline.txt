// https://www.veripool.org/papers/Preproc_Good_Evil_SNUGBos10_paper.pdf
`define MULTILINE line1 \
                 line2
`define MSG(m) `"m`"

$display(`MSG(`MULTILINE))