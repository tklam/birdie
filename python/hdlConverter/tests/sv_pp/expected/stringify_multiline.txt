// https://www.veripool.org/papers/Preproc_Good_Evil_SNUGBos10_paper.pdf




$display("line1 \
                 line2")